substrate/frame/examples/frame-crate/src/lib.rs

```diff
@@ -0,0 +1,66 @@
+// This file is part of Substrate.
```

Comment:
Is the name of this example `frame-crate` the best? Shouldn't we call it `frame-pallet`? Crate could refer to any Rust crate, not just a FRAME pallet.

Reply:
This is definitely not a pallet, maybe we could call it `frame-umbrella` instead?

Reply:
Oh, I got confused  :) `frame-crate` probably makes the most sense here.